README.md

```diff
@@ -12,6 +12,7 @@
         * [AWS IoT Device Shadow](#aws-iot-device-shadow)
         * [AWS IoT Jobs](#aws-iot-jobs)
         * [AWS IoT Device Defender](#aws-iot-device-defender)
+        * [AWS IoT Over-the-air Updates](#aws-iot-over-the-air-updates)
```

Comment:
Add "(Release Candidate)"

Reply:
Addressed in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1454/commits/aa538749dedbcaddc4d3c7b0179ef4d1ce21392d